tox.ini

```diff
@@ -61,23 +62,30 @@ deps =
     -r requirements-lint.txt
 description = Check code against coding style standards
 
-[testenv:kubeflow]
+[testenv:kubeflow-{local,remote}]
 commands =
     # run all tests apart from the ones that use MLFlow
     pytest -vv --tb native {[vars]driver_path} -s --filter "not mlflow" --model kubeflow {posargs}
+setenv =
+    local: LOCAL = "True"
+    remote: LOCAL = "False"
 deps =
     -r requirements.txt
 description = Run UATs for Kubeflow
 
-[testenv:uats]
+[testenv:uats-{local,remote}]
 # provide a filter when calling tox to (de)select test cases based on their names, e.g.
 # * run all tests containing 'kfp' or 'katib' in their name:
 #   $ tox -e uats -- --filter "kfp or katib"
 # * run any test that doesn't contain 'kserve' in its name:
 #   $ tox -e uats -- --filter "not kserve"
 # this simulates the behaviour of running 'pytest -k "<filter>"' directly on the test suite:
 # https://docs.pytest.org/en/7.4.x/reference/reference.html#command-line-flags
-commands = pytest -vv --tb native {[vars]driver_path} -s --model kubeflow {posargs}
+commands = 
+    pytest -vv --tb native {[vars]driver_path} -s --model kubeflow {posargs}
+setenv =
+    local: LOCAL = "True"
+    remote: LOCAL = "False"
```

Comment:
I will not block on this, but Is there a reason to surround the envs with quotes and then remove them in the `test_kubeflow_workloads.py`?

Reply:
I tried removing them but it didn't help. Surrounding them or not still resulted in them having the quotes, since they are string envs (I guess).

Reply:
we discussed and some changes were pushed, thanks @orfeas-k